src/components/TopNavDropdown/TopNavButton.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <button class="type-small-700 text-gray-500 whitespace-nowrap h-[76px] px-8 hover:bg-gray-50 focus:outline-black focus:outline-dashed active:bg-black active:!text-white">
```

Comment:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/83967528/210587431-d74dc62f-4661-4b07-8be8-bfc77450361a.png">

isn't this too high?

Reply:
what is too high? what part exactly

Reply:
The `Products` button. The gray area.

Reply:
not sure if you can access this but it is big by design, it is supposed to be full to the height of that nav
https://www.figma.com/proto/DemxrOr7KWj78cfNVoUw7N/Vision-Design-System?page-id=556%3A4880&node-id=659%3A6728&viewport=472%2C478%2C0.45&scaling=min-zoom

Reply:
[and on the zeplin](https://app.zeplin.io/project/6357fb022dba82821ad857bb/screen/63939138272909bc462a2bfa)